src/store/RepoStore/RepoStore.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  GetOrganisationRepoDataParams,
+  // eslint-disable-next-line prettier/prettier
+  IGitHubStore
+} from '@entities/githubstore/client';
+import { IRepo } from '@entities/repos/client';
+import { Meta } from '@utils/meta';
+import {
+  action,
+  computed,
+  makeObservable,
+  observable,
+  // eslint-disable-next-line prettier/prettier
+  runInAction
+} from 'mobx';
+
+import ApiStore from '../ApiStore/ApiStore';
+
+const BASE_URL = 'https://api.github.com';
+
+type PrivateFields = '_meta' | '_repo' | '_readme';
+
+export default class RepoStore implements IGitHubStore {
+  private readonly _apiStore = new ApiStore(BASE_URL);
+  private _meta: Meta = Meta.initial;
+  private _repo: IRepo = {} as IRepo;
+  private _readme: string = '';
+
+  constructor() {
+    makeObservable<RepoStore, PrivateFields>(this, {
+      _meta: observable,
+      _repo: observable.ref,
+      _readme: observable.ref,
+      meta: computed,
+      getOrganizationRepoData: action,
```

Comment:
поскольку используешь не стрелочную функцию, нужно делать проставлять ей контекст 
getOrganizationRepoData: action.bound
```suggestion
      getOrganizationRepoData: action.bound,
```